internal/node.bzl

```diff
@@ -117,20 +123,38 @@ _NODEJS_EXECUTABLE_ATTRS = {
 }
 
 _NODEJS_EXECUTABLE_OUTPUTS = {
-    "loader": "%{name}_loader.js"
+    "loader": "%{name}_loader.js",
+    "main": "%{name}_launcher.sh"
 }
 
-nodejs_binary = rule(
+nodejs_binary_rule = rule(
```

Comment:
users will see this name in `bazel query --output=label_kind` so we should make this name match the name they use